integration/simulation/simulation.go

```diff
@@ -42,8 +42,8 @@ type Simulation struct {
 	SimulationTime   time.Duration
 	Stats            *stats.Stats
 	Params           *params.SimParams
-	LogChannel       chan types.Log
-	Subscriptions    []ethereum.Subscription
+	LogChannels      map[string]chan types.Log // Maps an owner to the channel on which they receive logs for all their wallets.
```

Comment:
Maintaining an owner -> channel mapping will be necessary once we start checking whether the node only receives relevant logs (this isn't possible currently, as we are missing the filtering by sub ID on the host side).